web-animations-2/Overview.bs

```diff
@@ -2937,6 +2937,27 @@ new SequenceEffect(
 
 </div>
 
+<h4 id="keyframe-offset-type">Modification to the <code>*Keyframe</code> dictionaries</h4>
+
+The <code>double?</code> type of the <code>offset</code> key
+in <code>ComputedKeyframe</code>,
+<code>BaseComputedKeyframe</code>,
+<code>BaseKeyframe</code>,
+<code>BasePropertyIndexedKeyframe</code>,
+etc.
+is replaced by <code>(CSSNumberish? or TimelineRangeOffset or DOMString)</code>,
+where <code>double</code> continues to be interpreted as a percentage,
+a {{DOMString}} is parsed into a {{TimelineRangeOffset}}
+according to the syntax of the ''@keyframes'' prelude,
```

Comment:
Is there a link to the syntax of the @​keyframes prelude so I can check what that is?

Reply:
Replaced with `<keyframe-selector>`, which is more precise. :)

Reply:
Thank you!